lib/Form.tsx

```diff
@@ -21,21 +31,24 @@ export interface SubmitHandler<T = FormContent> {
   (data: T, helpers: Helpers): void;
 }
 
-interface Props {
+interface Props extends Omit<FormProps, "onSubmit"> {
   initialData?: object;
   children: React.ReactNode;
   context?: Context;
   schema?: ObjectSchema<object>;
   onSubmit: SubmitHandler;
 }
 
-export default function Form({
-  initialData = {},
-  children,
-  schema,
-  context = {},
-  onSubmit
-}: Props) {
+export default function Form(props: Props) {
+  const {
+    initialData = {},
+    children,
+    schema,
+    context = {},
+    onSubmit,
+    ...restProps
```

Comment:
Not a big deal, but here I think it would be better if keep the pattern we're using in the rest of the project.

`...restProps` -> `...rest`

Reply:
Ok, I think you are right. I'm going to change it.